src/cfnlint/data/ExtendedSpecs/all/04_property_values.json

```diff
@@ -1223,6 +1223,13 @@
       "ValueType": "VpcId"
     }
   },
+  {
+    "op": "add",
+    "path": "/ResourceTypes/AWS::EC2::SecurityGroup/Properties/GroupDescription/Value",
+    "value": {
+      "Pattern": "^([a-z,A-Z,0-9,. _\\-:/()#,@[\\]+=&;\\{\\}!$*])*$"
```

Comment:
My only real debate here is if should keep this to a ValueType.  Do you think there is a chance that certain things will always follow a pattern that we could reuse across many different things?  For instance with Security Groups we could put the pattern with the Security Group of `sg-someregex` then as we associate to that we can regex the values if they are provided.    In this case it is probably pretty particular to the Group Description attribute... not sure if we would run across this regex again.

Reply:
Added in another example that is interesting for this discussion, see the new commit

Reply:
I like having the regex under the Valut Type fields to reduce repeating.  What do you think?

Reply:
Well, at first I thought: "overkill". Now that a SecurityGroup CidrIp is popping up 4 times already, I might just agree...

Makes it also possible to (idea) add a Human Readable error. Regex might not always be "user friendly"

Reply:
I'll update this PR soon with these changes